src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
Updated to include subheaders and app filters.

![screen shot 2015-02-17 at 8 49 37 pm](https://cloud.githubusercontent.com/assets/23885/6242144/82f7b8bc-b6e6-11e4-82e3-b5d9937bf47d.png)